pennylane/qchem/convert_openfermion.py

```diff
@@ -13,19 +13,90 @@
 # limitations under the License.
 """
 This module contains the functions for converting an openfermion fermionic operator to Pennylane
-FermiWord or FermiSentence operators.
+Sum, LinearCombination, FermiWord or FermiSentence operators.
 """
+
+from functools import singledispatch
+from typing import Union
+
 # pylint: disable= import-outside-toplevel,no-member,unused-import
-from pennylane.fermi import FermiSentence, FermiWord
+import pennylane as qml
+from pennylane import numpy as np
+from pennylane.fermi.fermionic import FermiSentence, FermiWord
+from pennylane.ops import LinearCombination, Sum
+from pennylane.qchem.convert import (
+    _openfermion_to_pennylane,
+    _pennylane_to_openfermion,
+    _process_wires,
+)
+from pennylane.wires import Wires
+
+
+def _import_of():
+    """Import openfermion."""
+    try:
+        # pylint: disable=import-outside-toplevel, unused-import, multiple-imports
+        import openfermion
+    except ImportError as Error:
+        raise ImportError(
+            "This feature requires openfermion. "
+            "It can be installed with: pip install openfermion."
+        ) from Error
+
+    return openfermion
+
+
+def from_openfermion_qubit(of_op, tol=1.0e-16, **kwargs):
```

Comment:
Please make this function private for now. I will unify the qubit and fermionic support in a future PR.